modules/oxide/src/less/theme/components/toolbar-button/toolbar-number-input.less

```diff
@@ -0,0 +1,27 @@
+//
+// Toolbar number input
+//
+
+.tox {
+  .tox-number-input {
+    display: flex;
+
+    :focus {
+      border: 2px solid #2d6adf;
+      border-radius: 3px;
+    }
+  }
+
+  .tox-number-input > button {
+    background: #f7f7f7;
```

Comment:
These needs to be variables if you compare it with other components it's variables that are unique for the component but then use more generic variables as it's values. So any color, border width, width etc needs to be a variable.

Reply:
done